Dockerfile

```diff
@@ -1,10 +1,11 @@
-FROM golang:1.11.5 AS builder
-ADD . /src
-RUN cd /src \
- && go test . \
- && go build -o binary
+FROM golang:alpine as builder
+RUN apk add git && \
+	go get -d -v github.com/terorie/od-database-crawler
```

Comment:
Can we add from the local FS instead?
This always pulls from master and won't work for PRs/branches.

Reply:
Also, please compile with `-ldflags="-s -w"`

Reply:
Once it has been pushed to hub.docker.com it would need to pull from a remote (this repo) and then pull what it needs, if you set it to a local path when its running the build on dockers  end it  wont know where to pull data from as it only reads the docker file

Reply:
No, I'm using the Docker Hub GitHub Bot, which has access to the files from `COPY . /path`, see:
https://cloud.docker.com/repository/docker/terorie/od-database-crawler/builds/38708ae7-d109-4794-95c1-e59474b84ac8

Reply:
Pushed